semcore/dropdown-menu/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [4.21.2] - 2024-03-01
+
+### Fixed
+
+- selection item by keypress `Space` for button trigger.
```

Comment:
Do you mean that it's fixed when button trigger "is used"?

Reply:
yes, I'll rewrite)